envs/uwsgi-env.yaml

```diff
@@ -6,9 +6,9 @@ packages:
     git_tag: 5c2760222698c28ecf9d5608f94d9cdcaa2d9cb8  #2.13.1
     patches:
       - feedstock-patches/jansson/0001-Open-CE-changes.patch
+{% endif %}
```

Comment:
Is jansson available on s390x?

Reply:
Yes. defaults channel has jansson package for s390x.

Reply:
Great!